manage.py

```diff
@@ -205,6 +206,20 @@ def validate_schema(ctx, param, value):
     return schema
 
 
+def construct_extra_where_clause(cursor, filter_field, filter_value):
+    """
+    Returns part of a sql where clause, for given filter parameters.
+
+    :param str cursor: a pschopg2 database cursor
+    :param str filter_field: a period seperated field name, e.g. "tender.procurementMethod"
+    :param str filter_value: the value of the specified field, e.g. "direct"
+    """
+    path = [f"'{x}'" for x in filter_field.split('.')]
+    sql = 'AND d.data->' + '%s->'*(len(path)-1) + '>%s = %s'
+    escaped_sql = cursor.mogrify(sql, filter_field.split('.') + [filter_value])
```

Comment:
The variables `sql` and `escaped_sql` would make more sense as `format_string` and `where_fragment`.